plat/xilinx/common/ipi.c

```diff
@@ -0,0 +1,215 @@
+/*
+ * Copyright (c) 2017-18, ARM Limited and Contributors. All rights reserved.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+/*
+ * Xilinx IPI agent registers access management
+ */
+
+#include <errno.h>
+#include <ipi.h>
```

Comment:
First C includes, then TF includes, then platform includes:

```
#include <errno.h>
#include <string.h>

#include <common/debug.h>
#include <common/runtime_svc.h>
#include <lib/bakery_lock.h>
#include <lib/mmio.h>

#include <ipi.h>
#include <plat_ipi.h>
#include <plat_private.h>
```

Same for the other files.